app/controllers/annotations_controller.rb

```diff
@@ -74,15 +75,16 @@ def admin_update
     @section = @question.section
     @phase = @section.phase
     if ex_save && guid_save
-      redirect_to admin_show_phase_path(id: @phase.id, section_id: @section.id, question_id: @question.id, edit: 'true'), notice: _('Information was successfully updated.')
+      typ = (!ex_save && !guid_save ? 'example answer and guidance' : (!guid_save ? 'guidance' : 'example answer'))
```

Comment:
Yes indeed and now that I'm looking at it 'example answer and guidance' should probably be in a `_('...')` fastgettext call